doc/getting_started.rst

```diff
@@ -117,6 +117,17 @@ the test data::
   >>> accuracy_score(pipe.predict(X_test), y_test)
   0.97...
 
+.. note:: Diagram rendering of estimators
+
+    To render estimators as diagrams in notebooks, use the `display='diagram'` option:
+    >>> with config_context(display='diagram'):
+    >>>   pipe
+
+    You may also call `set_config 
+    <https://scikit-learn.org/stable/modules/generated/sklearn.set_config.html>`_
```

Comment:
please use a `:func:` sphinx reference instead.

Also above we'll need an import for `config_context`.

Reply:
When I run these lines in Jupyter:
```python
import sklearn
from sklearn import set_config

with sklearn.config_context(display='diagram'):
    pipe
```

I don't get any output (no errors either).

Reply:
ah, the context manager is such that the cell doesn't return the estimator. That's unfortunate, I wanted to avoid that all subsequent estimator would be rendered as diagrams but that won't be possible.

In this case, I would just remove the note and call `set_config` at the beginning of the pipeline section code blcok, with a short comment like:

```py
set_config(display='diagram')  # for HTML repr of estimators
```

Reply:
What if the code was replaced with what is below?  This does render the diagram in Jupyter notebook.

```python
from sklearn import set_config
set_config(display='diagram')

pipe
```

Reply:
Sure, we can keep the note